jumanji/environments/routing/connector/env.py

```diff
@@ -40,24 +40,23 @@
     is_valid_position,
     move_agent,
     move_position,
-    switch_perspective,
 )
 from jumanji.environments.routing.connector.viewer import ConnectorViewer
 from jumanji.types import TimeStep, restart, termination, transition
 from jumanji.viewer import Viewer
 
 
 class Connector(Environment[State]):
-    """The `Connector` environment is a multi-agent gridworld problem where each agent must connect a
-    start to a target. However, when moving through this gridworld the agent leaves an impassable
-    trail behind it. Therefore, agents must connect to their targets without overlapping the routes
-    taken by any other agent.
+    """The `Connector` environment is a gridworld problem where multiple pairs of points (sets)
+    must be connected without overlapping the paths taken by any other set. This is achieved
+    by allowing certain points to move to an adjacent cell at each step. However, each time a
+    point moves it leaves an impassable trail behind it. The goal is to connect all sets.
 
     - observation - `Observation`
         - action mask: jax array (bool) of shape (num_agents, 5).
         - step_count: jax array (int32) of shape ()
             the current episode step.
-        - grid: jax array (int32) of shape (num_agents, size, size)
+        - grid: jax array (int32) of shape (size, size)
             - each 2d array (size, size) along axis 0 is the agent's local observation.
```

Comment:
We have used `size` here but `grid_size` elsewhere - could we choose one and stick with it for consistency?